routes/index.js

```diff
@@ -1,34 +1,51 @@
-const db = require("../database/db");
+const db = require("../database/db")
+const programFile = require("./program")
+const applicationFile = require("./application")
+const program_db = require("../database/program_db")
 
 module.exports = {
 	getHomePage: async function (request, response) {
-		let studentList = await db.getStudentsList();
-		let activeStudents = [];
-		for (let i = 0; i < studentList.length; i++) {
-			if (studentList[i].status == "active") {
-				activeStudents.push(studentList[i]);
-			}
+		let activeStudents = programFile.activeStudents(await db.getStudentsList()) 
+		activeStudents.sort( (a, b) => a.first_name.localeCompare(b.first_name, 'fr', {
```

Comment:
good idea to sort these... but it looks like you're sorting using french instead of english